### ISSUE TITLE
Conductor: Processing `blender_verification_request` from work queue

### ISSUE THREAD

**Issue by cameel** [golemfactory/concent#224]
This task depends on #207.

### Sequence of operation
Implement the following part of the #193 blueprint:
- Conductor (worker) receives `blender_verification_request` message from work queue
- The app creates a new instance of `VerificationRequest` in the database and a `BlenderSubtaskDefinition` instance associated with it.
    - If there are already `UploadReport`s corresponding to some files, the app links them with the `VerificationRequest` by setting the value of the foreign key in `UploadReport`.
- The app checks if files indicated by `source_package_path` and `result_package_path` in the `VerificationRequest` have reports.
- If all expected files have been uploaded, the app sends `blender_verification_order` task to the work queue.

### Task separation
We want to be able to connect three separate workers types to the queue: Concent, Conductor and Verifier. Each of them should have its own task handlers. When you start one of them with `celery worker` command it must not start processing tasks that belong to other worker types.

We'll achieve this by adding a new value to `settings.CONCENT_FEATURES`. Worker's tasks handlers should be registered only if that setting contains `'conductor-worker'` string.

### Setup for local development
For local development you'll need to run a RabbitMQ server and a single Celery worker. You can run RabbitMQ in a docker container in one terminal and start the worker in the other. See https://github.com/golemfactory/concent/issues/197#issuecomment-375620295. It should work out of the box because `settings/development.py` is configured to connect to a RabbitMQ server on default port running on localhost.

**Comment by cameel** [comment]
Update: models have changed in the blueprint:
- `UploadRequest` has been removed.
- `VerificationRequest` has different fields.
- `BlenderSubtaskDefinition` has been extracted from `VerificationReqest`.
- `verification_order` has been renamed to `blender_verification_order`.
- Many fields have been removed or modified in `blender_verification_order`.

**Comment by cameel** [comment]
Update: Celery tasks handlers should be registered only if `settings.CONCENT_FEATURES` contains `'conductor-worker'` string.

**Comment by rwrzesien** [comment]
@cameel One thing where implementation differs from description:

> Worker's tasks handlers should be registered only if that setting contains 'conductor-worker' string.

Currently `AVAILABLE_CONCENT_FEATURES` is OrderedDict with this structure:
```
    ("conductor-urls", {
        "required_django_apps": [
            "conductor",
        ],
        "url_patterns":         [
            url(r'^conductor/', include(conductor.urls, namespace='conductor')),
        ],
    }),
```

While celery custom task loader do this:
```
# Load tasks only from features specified in CONCENT_FEATURES variable
for feature_name in settings.CONCENT_FEATURES:
    app.conf.imports += tuple(
        ['{}.tasks'.format(app)
         for app in AVAILABLE_CONCENT_FEATURES[feature_name]['required_django_apps']
         if importlib.util.find_spec('{}.tasks'.format(app)) is not None]
    )
```

So actually it will always check what is in `"required_django_apps"` and if this app has `tasks.py` file. So when `"conductor-urls"` is in `CONCENT_FEATURES` it will load tasks too, so there is no need to separate this. But if it is required to separate this, I will have to modify how `AVAILABLE_CONCENT_FEATURES` structure looks like.
